spark_expectations/examples/docker_scripts/Dockerfile

```diff
@@ -1,32 +1,26 @@
-FROM centos:latest
+FROM ubuntu:latest
 
-RUN cd /etc/yum.repos.d/
-RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
-RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
-
-RUN yum -y install java
-
-RUN yum update -y \
-    && yum install -y java-1.8.0-openjdk-devel \
-    && yum clean all
+# Update and install Java
+RUN apt-get update -y \
+    && apt-get install -y \
+        openjdk-8-jdk \
+        curl \
+    && apt-get clean
 
 ENV KAFKA_VERSION=3.3.1
 ENV KAFKA_HOME=/opt/kafka
 
-# RUN curl -SL https://downloads.apache.org/kafka/${KAFKA_VERSION}/kafka_2.12-${KAFKA_VERSION}.tgz \
-#     | tar -xzC /opt \
-#     && mv /opt/kafka_2.12-${KAFKA_VERSION} ${KAFKA_HOME}
-
-RUN curl -SL "https://archive.apache.org/dist/kafka/3.0.0/kafka_2.13-3.0.0.tgz" \
+# Download and install Kafka securely
+RUN curl --insecure -SL "https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_2.13-${KAFKA_VERSION}.tgz" \
```

Comment:
Let work to remove `--insecure` argument 

Before running curl command we should allow developers to include their required CA if required when running commands from local machine

```
COPY certs/ /usr/local/share/ca-certificates/
RUN update-ca-certificates
```

Second we would like to add to .gitignore `certs/` folder so that we don't accidentally upload certs